test/cypress/support/index.js

```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
```

Comment:
I think this is also an example file that we don't need right?

Reply:
No, it's the central import for support file, such as `commands.js` below.